### PR TITLE
Fixed a few mispellings

### PR DESCRIPTION
**Pull request by RemarkableWhiteboard**
Fixed 2 minor misspellings of words in data>magicitems>scroll.data

### CODE REVIEW

**Comment by RemarkableWhiteboard** [comment]
Fixed some typos/grammar errors

File: data/magicitems/scroll.data [hunk 1]
```diff
@@ -458,11 +458,11 @@ HSET scroll_effect_description shrinking    {"name":"shrinking",    "description
...
 HSET scroll_effect_description sickness    {"name":"sickness",    "description":"causes the user to display symptoms of severe illness" }
 LPUSH scroll_effect sickness
-HSET scroll_effect_description skelegro    {"name":"skelegro",    "description":"regrows any missing bones" }
-LPUSH scroll_effect skelegro
+HSET scroll_effect_description skelegrow    {"name":"skelegrow",    "description":"regrows any missing bones" }
+LPUSH scroll_effect skelegrow
 HSET scroll_effect_description slashingfury    {"name":"slashing fury",    "description":"causes the user to fly into a frenzy when holding a blade" }
 LPUSH scroll_effect slashingfury
-HSET scroll_effect_description slaying    {"name":"slaying",    "description":"grants the user unnatural vigor when combating a specific enemy" }
+HSET scroll_effect_description slaying    {"name":"slaying",    "description":"grants the user unnatural vigor when combatting a specific enemy" }
```
Comment (morgajel) [inline data/magicitems/scroll.data]: Sorry, just now getting around to this. First off, thank you for the fixes! I have just a few small feedback notes.

First up, Combating is the more comon form:
https://books.google.com/ngrams/graph?content=combatting%2C+combating&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Ccombatting%3B%2Cc0%3B.t1%3B%2Ccombating%3B%2Cc0

File: data/motivations/follow.data [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 LPUSH motivationfollow_text to follow others
 LPUSH motivationfollow_text to investigate rumors of riches
-LPUSH motivationfollow_text to learned more about rediscovered ancient lore
-LPUSH motivationfollow_text to follow the voices
```
Comment (morgajel) [inline data/motivations/follow.data]: perhaps "the voices" should be in quotes, or refer to the voices in their head.

File: data/npcphobias.data [hunk 3]
```diff
@@ -4,8 +4,8 @@ SET npc_phobia_chance 20
 LPUSH npc_phobia accidents ( Dystychiphobia )
```
Comment (morgajel) [inline data/npcphobias.data]: This file kills me... I've actually killed it in place of another file in a phobia branch. I just merged it in today. :/

**Comment by morgajel** [comment]
I ended up manually pulling in some of the corrections- thank you for your contribution!

**Pull request closed without merging**